### DISCUSSION TITLE
I could not install the app on the watch

### DISCUSSION THREAD

**Discussion by abutmim** [NightscoutFoundation/xDrip#2561] [Q&A]
Hi

I could not install the app on the watch

Samsung Note 10 lite
Huawei watch 2

![wear01](https://user-images.githubusercontent.com/40410642/208169997-74412823-daaa-4ffe-9aed-dfebfe04b146.jpg)

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
I've seen many conplaining about this recently. It's usually much easier to reset the watch and immediately install xDrip+ using the watch Google Play store, apps on your phone. But prior to this you need to confirm this is the official xDrip+ version.

**Reply by abutmim** [reply]
But prior to this you need to confirm this is the official xDrip+ version.

how can I do this?

I downloaded xDrip+ from here:
https://github.com/NightscoutFoundation/xDrip/releases

**Reply by psonnera** [reply]
Then it is, indeed, and it should include the Wear extension. Did you also try with the old Wear Installer app (not the 2)?

**Reply by abutmim** [reply]
Yes I tried but the same issue
